vscode-eslint/.gitignore

```diff
@@ -0,0 +1 @@
+/checkout/
```

Comment:
I suggest also add

```
/out/*.map
```

Reply:
Maybe also change

```diff
-SRC_DIR="${REPO_DIR}/checkout"
+SRC_DIR="${REPO_DIR}/src"
```

if this make more sense now. (also the .gitignore)

Reply:
Changed to `src`. As far as maps, I'd like to keep them as I found them useful on occasions when debugging eslint server.